app/src/main/java/org/literacyapp/receiver/BootReceiver.java

```diff
@@ -32,5 +35,16 @@ public void onReceive(Context context, Intent intent) {
         JobInfo jobInfo = builder.build();
         JobScheduler jobScheduler = (JobScheduler) context.getSystemService(Context.JOB_SCHEDULER_SERVICE);
         jobScheduler.schedule(jobInfo);
+        // Start service for detecting when the screen is turned on
+        Intent screenOnServiceIntent = new Intent(context, ScreenOnService.class);
+        context.startService(screenOnServiceIntent);
+
+        // Initiate background job for face recognition training
+        ComponentName componentNameFaceRecognitionTranining = new ComponentName(context, FaceRecognitionTrainingJobService.class);
+        JobInfo.Builder builderFaceRecognitionTranining = new JobInfo.Builder(FACE_RECOGNITION_TRAINING_JOB_SERVICE, componentNameFaceRecognitionTranining);
+        builderFaceRecognitionTranining.setPeriodic(15 * 60 * 1000); // Every 15 minutes
```

Comment:
As mentioned above, will this job take longer and longer time as more and more StudentImages are saved in the database, and some of them are not usable for feature extraction?